src/storage/config.rs

```diff
@@ -107,18 +107,14 @@ pub struct Config {
     #[online_config(skip)]
     pub txn_status_cache_capacity: usize,
     pub memory_quota: ReadableSize,
-    /// Maximum max_ts deviation allowed from PD timestamp
-    pub max_ts_drift_allowance: ReadableDuration,
-    /// How often to refresh the max_ts limit from PD
-    #[online_config(skip)]
-    pub max_ts_sync_interval: ReadableDuration,
-    pub action_on_invalid_max_ts: String,
```

Comment:
Are these configurations hidden from users?

Additionally, since the configurations have been changed, are there any compatibility issues that users might encounter during the upgrade?

Reply:
> Are these configurations hidden from users?

No they are visible to users.

>Additionally, since the configurations have been changed, are there any compatibility issues that users might encounter during the upgrade?

These items are added recently after v8.5 so no

Reply:
Got it, thx